.github/workflows/update-package-lock.yml

```
@@ -0,0 +1,29 @@
+name: Update package-lock.json
+on:
+  schedule:
+    - cron: "30 12 * * 1-5" # Mon-Fri 8:30AM EDT. 7:30AM EST.
+  workflow_dispatch: # manual trigger
+jobs:
+  update:
+    name: Update
+    timeout-minutes: 10
+    runs-on: [self-hosted, Linux, AWS]

```

Public repos can't run on our runners:

```suggestion
    runs-on: ubuntu-latest
```